cmd/fdsn-ws/fdsn_station.go

```diff
@@ -703,7 +700,10 @@ func (v fdsnStationV1Search) validBounding(latitude, longitude float64) bool {
 		return true
 	}
 
-	d, _ := geo.To(v.Latitude, v.Longitude, latitude, longitude)
+	d, _, err := wgs84.DistanceBearing(v.Latitude, v.Longitude, latitude, longitude)
+	if err != nil {
```

Comment:
sorry to be a pain - please could you at least log this error?  Otherwise we won't be able to trouble shoot if a request is not working for a user.

Reply:
Ok will do it.